docs/en/latest/plugins/zipkin.md

```diff
@@ -27,14 +27,15 @@ title: Zipkin
 - [**Attributes**](#attributes)
 - [**How To Enable**](#how-to-enable)
 - [**Test Plugin**](#test-plugin)
-  - [run the Zipkin instance](#run-the-zipkin-instance)
+  - [Run the Zipkin instance](#run-the-zipkin-instance)
+  - [Run the Jaeger instance](#run-the-jaeger-instance)
 - [**Disable Plugin**](#disable-plugin)
 
 ## Name
 
-[Zipkin](https://github.com/openzipkin/zipkin) is a OpenTracing plugin.
+[Zipkin](https://github.com/openzipkin/zipkin) an open source distributed tracing system. This plugin is supported to collect tracing and report to Zipkin Collector based on [Zipkin API specification](https://zipkin.io/pages/instrumenting.html).
 
-It's also works with `Apache SkyWalking`, which is support Zipkin v1/v2 format.
+It's also works with [Apache SkyWalking](https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/#zipkin-receiver) and [Jaeger](https://www.jaegertracing.io/docs/1.31/getting-started/#migrating-from-zipkin), which are support Zipkin [v1](https://zipkin.io/zipkin-api/zipkin-api.yaml)/[v2](https://zipkin.io/zipkin-api/zipkin2-api.yaml) format. And of course, it can  integrate other tracing systems adapted to Zipkin v1/v2 format as well.
```

Comment:
```suggestion
It's also works with [Apache SkyWalking](https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/#zipkin-receiver) and [Jaeger](https://www.jaegertracing.io/docs/1.31/getting-started/#migrating-from-zipkin), which are support Zipkin [v1](https://zipkin.io/zipkin-api/zipkin-api.yaml)/[v2](https://zipkin.io/zipkin-api/zipkin2-api.yaml) format. And of course, it can integrate other tracing systems adapted to Zipkin v1/v2 format as well.
```